.github/workflows/ci.yml

```diff
@@ -51,23 +52,50 @@ jobs:
             friendlyName: Windows
           - os: ubuntu-20.04
             friendlyName: Ubuntu
-        exclude:
-          - os: ubuntu-20.04
-            arch: arm64
+            image: ubuntu:18.04
     timeout-minutes: 60
     env:
       RELEASE_CHANNEL: ${{ inputs.environment }}
     steps:
+      - name: Install dependencies into dockerfile on Ubuntu
+        if: matrix.os == 'ubuntu-20.04'
+        run: |
+          # ubuntu dockerfile is very minimal (only 122 packages are installed)
+          # add dependencies expected by scripts
+          apt update
+          apt install -y software-properties-common lsb-release \
+          sudo wget curl build-essential jq autoconf automake \
+          pkg-config ca-certificates rpm \
+          # add electron dependencies
+          apt install -y libasound2 libatk-bridge2.0-0 libatk1.0-0 \
+          libatspi2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libdrm2 \
+          libexpat1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 \
+          libnss3 libpango-1.0-0 libx11-6 libxcb1 libxcomposite1 \
+          libxdamage1 libxext6 libxfixes3 libxkbcommon0 libxrandr2 \
+          libsecret-1-0
+          # install new enough git to run actions/checkout
+          sudo add-apt-repository ppa:git-core/ppa -y
+          sudo apt update
+          sudo apt install -y git
+          # avoid "fatal: detected dubious ownership in repository at '/__w/shiftkey/desktop'" error
+          git config --global --add safe.directory '*'
       - uses: actions/checkout@v3
         with:
           repository: ${{ inputs.repository || github.repository }}
           ref: ${{ inputs.ref }}
           submodules: recursive
       - name: Use Node.js ${{ matrix.node }}
+        if: matrix.os != 'ubuntu-20.04'
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
           cache: yarn
+      - name: Install unofficial-builds Node.js 18.16.1 on Ubuntu
+        if: matrix.os == 'ubuntu-20.04'
+        run: |
+          # this version supports older GLIBC (official builds required a minimum of GLIBC 2.28)
+          curl -sL 'https://unofficial-builds.nodejs.org/download/release/v18.16.1/node-v18.16.1-linux-x64-glibc-217.tar.xz' | xzcat | sudo tar -vx  --strip-components=1 -C /usr/local/
```

Comment:
:art: perhaps we could keep this version in sync with `${{ matrix.node }}` - I'm not too familiar with this repository

Reply:
right now we can't because 18.16.1 is the first version that started using the `linux-x64-glibc-217` naming convention while there are no windows builds for this version.

Reply:
solved in latest commits https://github.com/shiftkey/desktop/pull/897/commits/6f6d8ad7d140287e45c56b593ed2a359cdff5643

Reply:
solved differently in https://github.com/shiftkey/desktop/pull/897/commits/ad61ef12cb752545afcd425e9c736953d36e29c5 (since we don't want to break arm64 windows builds from working even if they do complete without error)